bigframes/session/__init__.py

```diff
@@ -1362,6 +1362,58 @@ def read_gbq_function(
             2    TestCad$123456Str
             dtype: string
 
+        Another use case is to define your own remote funtion (or UDFs) and use
+        it later. For example, define the remote function:
+
+            >>> @bpd.remote_function()
+            ... def tenfold(num: int) -> float:
+            ...     return num * 10
+
+        Then, read back the deployed BQ remote function:
+
+            >>> func_ref_0 = bpd.read_gbq_function(
```

Comment:
[nit] let's rename it to something that would look nicer in the public docs, say `tenfold` again.

Reply:
Done.